crates/libs/registry/src/key_iterator.rs

```diff
@@ -3,66 +3,59 @@ use super::*;
 /// An iterator of registry key names.
 pub struct KeyIterator<'a> {
     key: &'a Key,
-    range: core::ops::Range<usize>,
+    idx: u32,
```

Comment:
```suggestion
    index: u32,
```

No need to abbreviate.

Reply:
Changed.